src/Workspaces/CSharpTest/Formatting/FormattingTests.cs

```diff
@@ -5872,5 +5872,46 @@ static string F(int a, int b, int c)
     }
 }");
         }
+
+        [WorkItem(62)]
```

Comment:
Can we put the string #62 into the code?  That will make the code clickable for anyone looking at it on GitHub.